nx.json

```
@@ -47,5 +39,10 @@
       "style": "scss"
     }
   },
-  "defaultProject": "lumberjack-app"
+  "defaultProject": "lumberjack-app",
+  "targetDefaults": {
+    "build": {
+      "dependsOn": [{ "projects": "dependencies", target: "build" }]
```

issue: I forgot quote marks around `target`
```suggestion
      "dependsOn": [{ "projects": "dependencies", "target": "build" }]
```